jmix-gradle-plugin/src/main/groovy/io/jmix/gradle/EnhancingAction.groovy

```diff
@@ -71,6 +76,42 @@ class EnhancingAction implements Action<Task> {
         }
     }
 
+    protected void addAbsentEmptyStores(ClassesInfo classesInfo, SourceSet sourceSet, Project project) {
```

Comment:
Pls, add javadoc for this method. It's not fully clear from this method name:
* add where?
* what store is "empty"?
* absent "where"?